.github/workflows/rust.yml

```diff
@@ -75,11 +75,27 @@ jobs:
       uses: actions/checkout@v2
     - name: Test
       run: cargo test --all-features
+
```

Comment:
The diff makes is somewhat hard to see, but I simply made a second job that runs coverage